.vib/minio/goss/goss.yaml

```
@@ -0,0 +1,43 @@
+command:
+  mc-version-development:
+    exec: mc --version
+    exit-status: 0
+    stdout:
+      - "DEVELOPMENT"
+    stderr: [ ]
+    timeout: 20000 
+  mc-make-bucket:
+    exec: mc mb test
+    exit-status: 0
+    stdout:
+      - "Bucket created successfully `test`."
+    stderr: [ ]
+    timeout: 20000    
+  mc-list-buckets:
+    exec: mc ls
+    exit-status: 0
+    stdout:
+      - "bin"
+      - "licenses"
+    stderr: [ ]
+    timeout: 20000 
+  mc-ls-play:
+    exec: mc ls play
+    exit-status: 0
+    stderr: [ ]
+    timeout: 20000  
+  mc-which-minio:
```

Maybe a little too picky, but `which` is not really a `mc` command:

```suggestion
  which-minio:
```

---

Changed!